src/drivers/actuators/vertiq_io/module.yaml

```diff
@@ -0,0 +1,271 @@
+module_name: Vertiq IO
+serial_config:
+    - command: vertiq_io telemetry ${SERIAL_DEV}
+      port_config_param:
+        name: VERTIQ_IO_CFG
+        group: Vertiq IO
+
+actuator_output:
+  output_groups:
+    - param_prefix: VERTIQ_IO
+      group_label: 'ESCs'
+      channel_label: 'ESC'
+      standard_params:
+        min: { min: 0, max: 65535, default: 0 }
+        max: { min: 0, max: 65535, default: 65535 }
+        failsafe: { min: 0, max: 500 }
+      num_channels: 16
+
+parameters:
+    - group: Vertiq IO
+      definitions:
+        VERTIQ_BAUD:
+            description:
+                short: The IQUART driver's baud rate
+                long: |
+                    The baud rate (in bits per second) used by the serial port connected with IQUART communication
+            type: int32
+            default: 115200
+            reboot_required: true
+        VERTIQ_NUM_CVS:
+            description:
+              short: The number of Vertiq IFCI parameters to use
+              long: |
+                  The total number of IFCI control variables being used across all connected modules
+            type: int32
+            default: 0
+            min: 0
+            max: 16
+            reboot_required: true
+        DISARM_TRIGGER:
+            description:
+                short: The triggered behavior sent to the motors on PX4 disarm
+                long: |
+                    The behavior triggered when the flight controller disarms. You have the option to trigger your motors' disarm behaviors, set all motors to coast,
+                    or set a predefined throttle setpoint
+            type: enum
+            values:
+                0: Send Explicit Disarm
+                1: Coast Motors
+                2: Set Predefined Velocity Setpoint
```

Comment:
Why are you using a velocity setpoint instead of a throttle setpoint for this kind of behavior? All the control during flight and at all other times is in terms of throttle, which may or may not be turned to velocity based on your Mode. So why here do we change to specifying velocity? Why not Set Predefined Throttle?

Reply:
In response to myself here after talking with Jordan:

There are existing disarmed throttle value parameters in PX4, like PWM_MAIN_DIS1, that try to set the throttle value when you are disarmed. If setting that parameter works for the whole mixer, than we can just set that existing PX4 parameter appropriately and just send out whatever the mixer is sending as our disarmed value, instead of doing special stuff here to define our own predefined throttle setpoint. Maybe worth testing out

Also you wanted me to send you this to help you find stuff:
setAllDisarmedValues